cs/configuring.texy

```diff
@@ -107,6 +108,28 @@ $configurator->addParameters([
 ]);
 \--
 
+Lze definovat i dynamické parametry, které nezpůsobí přegenerování containeru.
```

Comment:
Ono to stále nevyjadřuje podstatu, co jsou to dynamické parametry. 

Asi bych na to šel tak, že normální parametry jsou ty, jejichž hodnota musí být známa v době kompilace (compile-time) a pro různé hodnoty se vytvářejí různé kontejnery, zatímco dynamické se mohou měnit s každým požadavkem (run-time). Takže první případ je třeba parametr určující, jaké služby kontainer vytváří, např. zda pro odesílání emailů používat SMTP nebo něco jiného, zatímco druhý případ je třeba IP adresa návštěvníka stránky.